app/models/request.rb

```diff
@@ -563,6 +563,11 @@ def product_line
   def manifest_processed!
   end
 
+  def self.get_all_comments(request)
+    counts = Comment.counts_for_requests([request])
+    counts[request.id]
+  end
```

Comment:
Handy that this `counts_for_requests` method existed already! Looks like that's used for a similar link on the Pipeline inbox page, e.g. here - https://training.sequencescape.psd.sanger.ac.uk/pipelines/51